explorer/client/src/components/search/Search.tsx

```diff
@@ -8,8 +8,22 @@ import { navigateWithUnknown } from '../../utils/searchUtil';
 
 import styles from './Search.module.css';
 
+type SearchCategory = 'all' | 'transactions' | 'addresses' | 'objects';
+function getPlaceholderText(category: SearchCategory) {
+    switch (category) {
+        case 'addresses':
+            return 'Search by address';
+        case 'transactions':
+            return 'Search by digest';
```

Comment:
```suggestion
            return 'Search by transaction id';
```
Even though "digest" is technically more accurate, we use `TxnId` everywhere else, so let's be consistent here
![CleanShot 2022-05-13 at 12 18 15](https://user-images.githubusercontent.com/76067158/168375071-4bc2d49c-6434-4cec-92d8-62cc87dd30a1.png)